apps/sms/test/marionette/generators/thread.js

```diff
@@ -59,8 +59,8 @@ var ThreadGenerator = {
 
     for (var i = 0; i < parameters.numberOfMessages; i++) {
       var message = null;
-      var messageBody = thread.body + ' (messageId: ' +
-        (ThreadGenerator.uniqueMessageId + 1) + ')';
+      var messageBody = thread.body +
+        (i ? ' (messageId: ' + ThreadGenerator.uniqueMessageId + ')' : '');
```

Comment:
I wanted to have the thread body for the most recent message; and then I removed "+1" but likely because I didn't really understood how this works. I'll put it back (keeping the condition though).